src/__tests__/CompareResults/Retrigger.test.tsx

```diff
@@ -0,0 +1,246 @@
+        name: 'retrigger-multiple',
+        symbol: 'rt',
+        taskGroupId: 'ZccXq7SFRLyRF7l6unyxkQ',
+        title: 'Retrigger',
+      },
+      push: {
+        base_revision: '51748d809eb5e6a7c86eeeb911f3b2b12324b1bf',
+        owner: 'mozilla-taskcluster-maintenance@mozilla.com',
+        pushlog_id: '42024',
+        revision: 'ab17014f4b8f237c9d5fe7b88f8a89c49d0ffe7c',
+      },
+      repository: {
+        level: '3',
+        project: 'mozilla-central',
+        url: 'https://hg.mozilla.org/mozilla-central',
+      },
+    },
+    user: {
+      input: {
+        $eval: 'input',
+      },
+      taskGroupId: {
+        $eval: 'taskGroupId',
+      },
+      taskId: {
+        $eval: 'taskId',
+      },
+    },
+  },
+  kind: 'hook',
+  name: 'retrigger-multiple',
+  schema: {
+    properties: {
+      additionalProperties: false,
+      requests: {
+        items: {
+          additionalProperties: false,
+          tasks: {
+            description: 'An array of task labels',
+            items: {
+              type: 'string',
+            },
+            type: 'array',
+          },
+          times: {
+            description: 'How many times to run each task.',
+            maximum: 100,
+            minimum: 1,
+            title: 'Times',
+            type: 'integer',
+          },
+        },
+        type: 'array',
+      },
+    },
+    type: 'object',
+  },
+  title: 'Retrigger',
+};
+
+const setUpUserCredentials = () => {
+  sessionStorage.setItem('requestState', 'OkCrH5isZncYqeJbRDelN');
+  sessionStorage.setItem(
+    'taskclusterUrl',
+    'https://firefox-ci-tc.services.mozilla.com',
+  );
+  window.localStorage.setItem(
+    'userTokens',
+    JSON.stringify({
+      'https://firefox-ci-tc.services.mozilla.com': {
+        access_token: 'RnVpOGJtdDZTb3FlWW5PVUxVclprQQ==',
+        token_type: 'Bearer',
+      },
+    }),
+  );
+  window.localStorage.setItem(
+    'userCredentials',
+    JSON.stringify({
+      'https://firefox-ci-tc.services.mozilla.com': {
+        expires: '2044-05-20T14:07:40.828Z',
+        credentials: {
+          clientId:
+            'mozilla-auth0/ad|Mozilla-LDAP|ldapuser/perfcompare-localhost-3000-client-OCvzh5',
+          accessToken: 'jQWJVQdeRceT-YymPwTWagPh2PwJr0RmmZyL1uAfMSWg',
+        },
+      },
+    }),
+  );
+};
+
+describe('Retrigger', () => {
+  beforeEach(() => {
+    mockedGetLocationOrigin.mockImplementation(() => 'http://localhost:3000');
+  });
+
+  it('should display Sign In modal when there are no credentials', async () => {
+    render(<RetriggerButton result={result} />);
+
+    const openModalButton = await screen.findByTitle('retrigger jobs');
+
+    const user = userEvent.setup({ delay: null });
+    await user.click(openModalButton);
+
+    const signInButton = await screen.findByText('Sign in');
+
+    expect(signInButton).toBeInTheDocument();
+  });
+
+  it('should display Retrigger modal when there are credentials', async () => {
```

Comment:
I think this test is not useful, the next test does the same thing. So I'd remove it.